packages/core/src/request.ts

```diff
@@ -0,0 +1,100 @@
+import { Status } from '@anticrm/status'
+import { Class, Doc, Ref } from './classes'
+import { DocumentQuery, Storage } from './storage'
+import { Tx } from './tx'
+
+export type ReqId = string | number
+
+export interface RequestStream {
+  send: (id: ReqId, method: string, params: any[]) => void
```

Comment:
send should accept `Request<T>` from `rpc`

Reply:
Fixed.